rollup/internal/controller/watcher/chunk_proposer.go

```diff
@@ -64,7 +65,7 @@ type ChunkProposer struct {
 }
 
 // NewChunkProposer creates a new ChunkProposer instance.
-func NewChunkProposer(ctx context.Context, cfg *config.ChunkProposerConfig, minCodecVersion encoding.CodecVersion, chainCfg *params.ChainConfig, db *gorm.DB, reg prometheus.Registerer) *ChunkProposer {
+func NewChunkProposer(ctx context.Context, cfg *config.ChunkProposerConfig, minCodecVersion encoding.CodecVersion, chainCfg *params.ChainConfig, l2BlockDB, db *gorm.DB, reg prometheus.Registerer) *ChunkProposer {
```

Comment:
ditto

Reply:
refactored in commit: [818add4](https://github.com/scroll-tech/scroll/pull/1645/commits/818add4fdc1052b92b070a4704e781a62e6db924).